types/adlib/index.d.ts

```diff
@@ -3,7 +3,7 @@
 // Definitions by: Esri <https://github.com/Esri>
 //                 Mike Tschudi <https://github.com/MikeTschudi>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.1
+// TypeScript Version: 3.9
```

Comment:
Is there a 3.9 requirement?  (This is usually used to limit the TS versions, so if there's no newer feature you need it's best to leave it.)

Reply:
Thank you; there isn't. Let's cancel this, please.